src/labkey/Experiment.ts

```diff
@@ -597,40 +597,64 @@ export function saveRuns(options: SaveRunsOptions): XMLHttpRequest {
     });
 }
 
-export interface GenIdActionsOptions extends RequestCallbackOptions {
+export interface EntitySequenceActionsOptions extends RequestCallbackOptions {
     containerPath?: string;
-    genId?: number;
-    kindName: 'SampleSet' | 'DataClass';
-    rowId: number;
+    seqType: 'genId' | 'rootSampleCount' | 'sampleCount';
```

Comment:
Is `containerPath` required when `seqType` is `rootSampleCount` or does it use the current container if not supplied. If it's the former, docs would be good.

Reply:
Ah, never mind. All of these are counters at the project level, right? So they all behave the same way with respect to `containerPath`.